sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1443,22 +1443,31 @@ export class ApplicationService extends RecordDataModelService<Application> {
   }
 
   /**
-   * Get enrolment details of an application.
-   * While fetching the enrolment details, the application
-   * expected to be in status Assessment or Enrolment or Completed.
+   * Get assessment details of an application.
    * @param applicationId application.
-   * @param studentId applicant student.
+   * @param options query options
+   * - `studentId` student id used for authorization.
+   * - `applicationStatus` application status to be considered.
```

Comment:
`application status/statuses to be considered`

Reply:
Changed.